CHANGELOG.md

```diff
@@ -16,6 +34,7 @@ This CHANGELOG follows the format listed [here](https://github.com/sensu-plugins
 ### Fixed
 - check-log.rb: pattern is now required via option config (@cgeers)
 - check-log.rb: update descriptions of crit and warn options (@cgeers)
+>>>>>>> upstream/master
```

Comment:
looks like missed from a merge conflict resolution.

Reply:
fixed in latest commit